go/apps/api/routes/v2_apis_list_keys/403_test.go

```diff
@@ -57,7 +57,7 @@ func TestAuthorizationErrors(t *testing.T) {
 		Name:        "Test API",
 		WorkspaceID: workspace.ID,
 		AuthType:    db.NullApisAuthType{Valid: true, ApisAuthType: db.ApisAuthTypeKey},
-		KeyAuthID:   sql.NullString{Valid: true, String: keyAuthID},
+		KeyAuthID:   sql.NullString{Valid: true, String: keySpaceID},
```

Comment:
ths one is missing

Reply:
actually there are plenty of KeyAuthID still in the db structs

what we should do is update the sql query to rename the struct field.

as an example, you can do something like this to rename it in go, but keep the column in the db untouched
 `WHERE key_auth_id = sqlc.arg(key_space_id)`

@Flo4604 is there a reason we chose not to do that?

Reply:
yeah, i don't think i touched the queries since I thought they mapped directly to the db columns, but I made the change without renaming the db columns and regenerated the sqlc